data/human/weapons.txt

```diff
@@ -11,6 +11,194 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Tethys Cannon"
+	category "Turrets"
+	cost 1067000
+	mass 128
+	thumbnail "outfit/rnd_gun2_turret"
+	"outfit space" -110
+	"weapon capacity" -97
```

Comment:
```suggestion
	mass 128
	thumbnail "outfit/rnd_gun2_turret"
	"outfit space" -128
	"weapon capacity" -128
```
Mass/Outfit/Weapon should generally be consistent unless there's a good reason to have them different. 
So pick a number and stick with it.